apps/src/javalab/playground/Playground.js

```diff
@@ -3,10 +3,10 @@ import {
   PlaygroundItemType,
```

Comment:
I just noticed that the links in the playground README need to be updated too. https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/javalab/playground/README.md